pygeoapi/api.py

```diff
@@ -1447,6 +1453,27 @@ def get_collection_items(
         else:
             skip_geometry = False
 
+        LOGGER.debug('processing filter parameter')
+        cql_text = request.params.get('filter')
+        if cql_text is not None:
+            try:
+                filter_ = parse_ecql_text(cql_text)
+            except Exception as err:
+                LOGGER.error(err)
+                msg = f'Bad CQL string : {cql_text}'
+                return self.get_exception(
+                    400, headers, request.format, 'InvalidParameterValue', msg)
+        else:
+            filter_ = None
+
+        LOGGER.debug('Processing filter-lang parameter')
+        filter_lang = request.params.get('filter-lang')
+        # Currently only cql-text is handled, but it is optional
+        if filter_lang is not None and filter_lang != 'cql-text':
```

Comment:
`if filter_lang not in [None, 'cql-text']:`

Reply:
Fixed in latest commits by @ximenesuk and @KoalaGeo.